databuilder/query_language.py

```diff
@@ -35,6 +35,9 @@ def __eq__(self, other):
     def __ne__(self, other):
         return BoolSeries(self._make_binary_fn(other, qm.Function.NE))
 
+    def isnull(self):
```

Comment:
It's surprising this isn't `is_null()`. I assume that is a deliberate choice. Can you explain it?

Reply:
User error, fixed.